server/middleware/form/__tests__/process.spec.js

```diff
@@ -429,10 +429,12 @@ describe('Process middleware', () => {
         expect(next).toHaveBeenCalledTimes(1);
     });
 
-    it('should not sanitise day and month with multiple connotations', () => {
```

Comment:
I don't believe this test is actually useful, we should just take what the user provides us and go with it.